src/Tasks/AlgoliaIndexTask.php

```diff
@@ -249,7 +260,12 @@ private function syncChanges($index) {
     }
 
     /**
-     * All pages that have been removed will
+     * remove all AlgoliaObjects of wich the page have been removed or ShowInSearch in the CMS has been set to false
+     *
+     * Get all the holders containing ID's of pages that have been removed,
+     * Get all the Page->ID's that have been synced and ShowInSearch have been toggled to false
```

Comment:
Get all the Page->ID's that have been synced and **where** ShowInSearch **has** been **changed** to false.